migrations/initial_migration_postgres.sql

```diff
@@ -0,0 +1,182 @@
+CREATE TABLE app_permissions (
+    id bigint NOT NULL,
+    app_id bigint,
+    request_method text,
+    max_amount bigint,
+    budget_renewal text,
+    expires_at timestamp with time zone,
+    created_at timestamp with time zone,
+    updated_at timestamp with time zone
+);
+
+CREATE SEQUENCE app_permissions_id_seq
```

Comment:
don't know what sequences are, do we need them here?

Reply:
it looks like it is postgres syntax for auto incrementing the id column? but it works, is it worth changing?

Reply:
Any "auto incrementing" primary key is a sequence. sequence can also be created individually. 

as you simply dumped the schema, I think it's fine and this is what PG does